### ISSUE TITLE
Installing SAP HANA 2.0, express edition (Virtual Machine Method)

### ISSUE THREAD

**Issue by murmelsson** [SAPDocuments/Tutorials#1965]
Took me some time to realise that the reason I was not getting an ipv4 address assigned from VMware, was that I needed to change the Network Adapter settings from the default after import (Bridged), to the NAT setting. Now everything seems to be working normally.

If this issue is common, might be worth adding a note in the Tutorial advising people to change to NAT?

**Comment by timemaster67** [comment]
however if you change your vm to nat, your host or any other program won't be able to connect to it, except if you fiddle with vmware nat port forwarding rules. That's why it's set to bridged by default, so that the vm get an ip address on the same network as your computer.

**Comment by murmelsson** [comment]
So if you are running a VM on VMware Player, with a laptop as host (by far the most typical use case for HANA-Express), then there is really no need to do any special Port Forwarding to be able to connect from the host to the VM. In addition, if you run 2 VMs as guests from VMWare Player (not everyone knows, but this is easily done using VMWare Player), then the guests can connect to each other using the IPs that VMware gave them on its "guest network"... so for example, say you have nginx running on Leap42.3 on VMware on 192.168.1.129; then in Ubuntu on vmware on 192.168.1.130, you can for sure go visit the welcome page on the nginx server. Whereas if you use Bridged, you might not get an ipv4 address at all, and then you are a bit stuck... but, on the other hand, Bridged is "nicer" because then, if you can get the VM to get an IP address, then it is more like an "equal citizen"-node, so might be easier e.g. to connect to it from your other laptop on the network... i still think the main use case is just access from the host laptop though, in that cae then NAT is sufficiently good.

**Comment by murmelsson** [comment]
Ok I realised the reason I wasn't getting an ipv4 address over Bridged... it is due to having a Windows 10 host that is using Direct Access to become an extension of my company network, and Teredo gives out no ipv4 address, neither to the host nor to any VMWare Bridging wannabe ipv4 node :) 

So a workaround for people who want to still use Bridged connection, is as follows (smoke-tested ok on my machine): Shut Down the VM. Go to the settings for Network... select (or keep) the Bridged option. press the button: Configure Adapters. Remove all troublsome adapters from the list, e.g. remove "Microsoft Wi-Fi Direct Virtual Adapter"... I kept only Intel(R) Ethernet Connection, and Intel(R) Dual Band Wireless...", obviously the specific adapter will depend on your own laptop and installed stuff etc... Save the settings, start the VM... Banzai!! The local DHCP server (in my case, a router at home), now happily assigns the VM an ipv4 address... now I can even SSH into the VM from an Android phone on the home network.

Overall then, Bridged Networking is probably ok, but in case it isn't, workarounds are either using NAT, or avoiding letting Direct Access get in the way of you getting an ipv4 address :)
